main/main.tex

```diff
@@ -54,113 +62,110 @@
 
 \maketitle
 
-Mass spectrometry (MS) is a versatile analytical technique that has been 
-adopted in a variety of disciplines, including proteomics, metabolomics, and 
-lipidomics. MS enables the identification and quantification of a wide range 
-of molecules. Obtaining high-quality data from mass spectrometry experiments 
-can be a challenging task, as numerous factors can impact the accuracy and 
-reproducibility of the obtained data. To ensure that MS data is of the 
-highest quality, it is imperative to implement appropriate quality control 
-(QC) techniques to guarantee that the data sets are reliable 
-\citep{Koecher2011}.
+Mass spectrometry (MS) is a versatile analytical technique that has been adopted in a
+variety of disciplines, including proteomics, metabolomics, and lipidomics. MS enables the
+identification and quantification of a wide range of molecules. Obtaining high-quality
+data from mass spectrometry experiments can be a challenging task, as numerous factors can
+impact the accuracy and reproducibility of the obtained data. To ensure that MS data is
+fit for purpose, quality assessment and quality control (QA/QC) need to be performed close
+% see e.g. Wikipedia for the difference between QA and QC in case it's not clear
+to data production \citep{Koecher2011}. Use of standardized quality metrics described by a
+controlled vocabulary helps in making QA/QC more comparable across datasets and data
+producers and increases transparency and trustworthiness of such measures as viewed by
+data users.
 
 Here, we introduce the \texttt{MsQuality} R-package, which provides 
 functionality to calculate, assess, and track quality metrics for mass 
-spectrometry-derived spectral data at the per-measurement level. 
-The package allows to compute about 40 of the mzQC quality metrics defined by
+spectrometry-derived spectral data at the per-measurement level. \fixme{define 'per-measurement level', otherwise this will mean many different things to different people}
+The package providse about\fixme{why 'about' and not the precise number?} 40 of the mzQC quality metrics defined by
 the Human Proteome Organization-Proteomics Standards Initiative (HUPO-PSI,
-hupo-psi.github.io/mzQC). These are calculated on low-level MS data
-such as retention times and m/z and associated intensity values.
+hupo-psi.github.io/mzQC).\cite{Also a paper citation?} These are calculated on low-level MS data
```

Comment:
To my knowledge the paper on mzQC is still in the making. @jorainer do you know of a resource that introduces this concept?

Reply:
No, not for *general* MS data - there is a QA working group for metabolomics, but they work on a different level (which control samples to include etc). I've also not seen the mzQC paper out yet.